src/rov/pca9685/CMakeLists.txt

```diff
@@ -23,21 +23,31 @@ find_package(ament_cmake REQUIRED)
 find_package(rov_interfaces REQUIRED)
 find_package(rclcpp REQUIRED)
 
-add_executable(pca9685_node src/PCA9685_Node.cpp src/JHPWMPCA9685.cpp src/JHPWMPCA9685.h src/ServoDriver.cpp src/ServoDriver.hpp)
-target_include_directories(pca9685_node PUBLIC
+add_library(pca9685 src/PCA9685_Node.cpp src/ServoDriver.cpp)
+add_subdirectory(external/JHPWMPCA9685)
```

Comment:
That is a *spicy* folder name. Is that something official, or can we make it less like... lajdflksajdflaj?

Reply:
Also look at you making a fancy library, so proud

Reply:
Also I have learned that yes, somebody came up with this name and it is legit, my apologies